src/game/action/action.h

```diff
@@ -33,7 +33,9 @@ enum class ActionType {
     StartConversation = 0x1001,
     PauseConversation = 0x1002,
     ResumeConversation = 0x1003,
-    MoveToObject = 0x1004
+    MoveToObject = 0x1004,
+
+    AttackAction
```

Comment:
To match _nwscript.nss_ we should name it `AttackObject` and give it a value of 3. This action could contain a type of attack in the future.